Source/Demos/Demo.Particles/Game1.cs

```diff
@@ -28,6 +28,7 @@ public Game1()
             Content.RootDirectory = "Content";
             IsMouseVisible = true;
             Window.AllowUserResizing = true;
+            Window.Position = Point.Zero;
```

Comment:
Just curious, why do you put the window at position zero? Is the default position a problem on Mac's?